Tests/Unit/Repository/RepositoryTestCase.php

```diff
@@ -11,10 +11,36 @@
 
 namespace Symfony\Cmf\Component\Resource\Tests\Unit\Repository;
 
-use Prophecy\PhpUnit\ProphecyTestCase;
+use Puli\Repository\Resource\Collection\ArrayResourceCollection;
+use Symfony\Cmf\Component\Resource\Repository\Resource\CmfResource;
 
-abstract class RepositoryTestCase extends ProphecyTestCase
+abstract class RepositoryTestCase extends \PHPUnit_Framework_TestCase
 {
+    protected $documentManager;
+    protected $managerRegistry;
+    protected $childrenCollection;
+    protected $finder;
+    protected $uow;
+    protected $document;
+    protected $child1;
+    protected $child2;
+    protected $object;
+    protected $resource;
+    protected $session;
+    protected $node;
+    protected $rootNode;
+
+    public function setUp()
+    {
+        $this->session = $this->prophesize('PHPCR\SessionInterface');
+        $this->finder = $this->prophesize('DTL\Glob\FinderInterface');
+        $this->node = $this->prophesize('PHPCR\NodeInterface');
+        $this->node2 = $this->getMock('PHPCR\NodeInterface');
```

Comment:
The expectected interface does not match when using ProphetObject. Got an error in the PhpcResource constructor and no match on instanceOf. I think it's is a commun usage in phpspec Not to mock interfaces using Test implementations instead. I did that for the document. But my node implementation can't be mocked as i got strange errors Last Night.